015/master

neugierige opened the pull request:
summary of changes:

1. addressed issue #2 
2. addressed issue #4 
3. localhost updated to 4741
4. in the code snippet for Book model, added "belong_to :author" to avoid confusion
5. "surname" has been updated to "family_name"
6. in curl POST for loan, added a missing "date" parameter
7. updated the variables in inverse_of example for better clarity
8. added questions to developers re: writing same curl command in Ruby (for both create and destroy)
9. updated diagram to match demo content
10. deleted section in clinic that adds patients and doctors because 015 used `examples.rb` to put doctors and patients in the database

Commit:
update as per issue #2

Because actual code is not written in this repo,
the standard "bundle install", add secrets, etc. process
does not apply.

Commit:
fixed typo as per issue #4

class Borrower has many `:books`, not `:borrowers`

Commit:
changed localhost from 3000 to 4741

Commit:
Book model retains relation to author

in code snippets, to avoid confusion, added "belong_to :author" back

Commit:
changed surname to family_name

Commit:
updates to POST/create a loan

1) added value to "date" parameter to CURL request
2) added question re: writing the same command in Rails console

Commit:
updated inverse_of example

1) changed variable name to be more explicit
2) changed "first_name" to "given_name"
3) delete duplicate of same code snippet in code-along, added prompt
to recall the example given already

Commit:
added prompt asking about Ruby equivalent of curl

for the Demo, under "Dependent Destroy"

Commit:
book-borrower-loan diagram added

to illustrate the many-to-many relationship the lesson is about
to demo

Commit:
updated link for diagram

from patient-physician-appointment to book-borrower-loan diagram

BenGitsCode commented:
Lu will be revising this tonight so that the starter code for this talk will match the examples Chris is using in his `rails-one-to-many` talk today.

Commit:
deleted "Adding Via ActiveRecord: Clinic" section

because in the 015 one-to-many talk, example.rb file has already
provided a list of patients and doctors

Commit:
updated text annotating the diagram

to align with new diagram for book-borrower-loan relationships

Commit:
updated lesson intro

1) added a description
2) added Objects
3) added Prerequisites
